tests/vmi_multus_test.go

```diff
@@ -928,7 +955,7 @@ func checkMacAddress(vmi *v1.VirtualMachineInstance, interfaceName, macAddress s
 	Expect(err).ToNot(HaveOccurred(), "MAC %q was not found in the VMI %s within the given timeout", macAddress, vmi.Name)
 }
 
-func pingVirtualMachine(vmi *v1.VirtualMachineInstance, ipAddr, prompt string) {
+func pingVirtualMachine(vmi *v1.VirtualMachineInstance, ipAddr, prompt string) error {
```

Comment:
This is a good opertunity to fix the confusing name which caused the error message to be wrong.
This is not ping-ing a VMI, but an address from a specific VMI. The address may be of a target VMI, but it does not have to be.

The `prompt` is also annoying, it is an implementation detail that the caller should not care really. It would have been nice to decode it from the `vmi` object itself, but I am not sure how easy it is. It we could have labeled or add a metadata to the VMI mentioning what OS it holds, it would have been easy.
(on the last point, I will be a nice to do but not a must here)